PlayCover/Views/MenuBarView.swift

```diff
@@ -96,14 +96,6 @@ struct PlayCoverViewMenuView: Commands {
             Button("menubar.clearCache") {
                 DataCache.instance.cleanAll()
                 URLCache.iconCache.removeAllCachedResponses()
-                do {
-                    let oldCacheFolder = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first!
```

Comment:
The reason why the old clear cache command did not work as intended is because it lacked two lines:
```swift
         .appendingPathComponent(Bundle.main.bundleIdentifier!)
         .appendingPathComponent("Image Cache")
```
Instead, this function is trying to remove the entire cache directory (`~/Library/Caches`). So, those two lines should be added at the end to make the function act as intended.